test/e2e/specs/delete-prohibited/recordset/facet-presentation.spec.ts

```diff
@@ -0,0 +1,249 @@
+  });
+
+  await test.step('should have ' + testParams.totalNumFacets + ' facets', async () => {
+    await expect.soft(RecordsetLocators.getAllFacets(page)).toHaveCount(testParams.totalNumFacets);
+
+    const titles = RecordsetLocators.getFacetTitles(page);
+    const count = await titles.count();
+    for (let i = 0; i < count; i++) {
+      await expect.soft(titles.nth(i)).toHaveText(testParams.facetNames[i])
+    }
+  });
+
+  await test.step('should have 3 facets open', async () => {
+    await expect.soft(RecordsetLocators.getOpenFacets(page)).toHaveCount(testParams.defaults.openFacetNames.length);
+
+    const titles = RecordsetLocators.getOpenFacetTitles(page);
+    const count = await titles.count();
+    for (let i = 0; i < count; i++) {
+      await expect.soft(titles.nth(i)).toHaveText(testParams.defaults.openFacetNames[i])
+    }
+  });
+
+  // test defaults and values shown
+  await test.step('"id" facet should have 1 facet option checked', async () => {
+    // use 0 index
+    const facet = RecordsetLocators.getFacetById(page, 0);
+    await expect.soft(RecordsetLocators.getFacetSpinner(facet)).not.toBeVisible();
+    await expect.soft(RecordsetLocators.getCheckedFacetOptions(facet)).toHaveCount(1);
+  });
+
+  await test.step('"int_col" facet should not show the histogram with 1 facet options checked', async () => {
+    // use 1 index
+    const facet = RecordsetLocators.getFacetById(page, 1);
+    await expect.soft(RecordsetLocators.getFacetSpinner(facet)).not.toBeVisible();
+    await expect.soft(RecordsetLocators.getFacetHistogram(facet)).not.toBeVisible();
+    await expect.soft(RecordsetLocators.getCheckedFacetOptions(facet)).toHaveCount(1);
+  });
+
+  await test.step('should have 2 filters selected', async () => {
+    await expect.soft(RecordsetLocators.getFacetFilters(page)).toHaveCount(testParams.defaults.numFilters);
+  });
+
+  await test.step('should have 1 row selected in show more popup for scalar picker and should be able to search in popup.', async () => {
+    const facet = RecordsetLocators.getFacetById(page, 0);
+
+    // open show more, verify only 1 row checked, check another and submit
+    await RecordsetLocators.getShowMore(facet).click();
+
+    const facetPopup = ModalLocators.getScalarPopup(page);
+    // one row is selected
+    await expect.soft(RecordsetLocators.getCheckedCheckboxInputs(facetPopup)).toHaveCount(1);
+
+    // search
+    const searchInp = RecordsetLocators.getMainSearchInput(facetPopup),
+      searchSubmitBtn = RecordsetLocators.getSearchSubmitButton(facetPopup);
+
+    await searchInp.fill('1|2');
+    await searchSubmitBtn.click();
+
+    const modalOptions = RecordsetLocators.getCheckboxInputs(facetPopup);
+    // make sure search result is displayed
+    await expect.soft(modalOptions).toHaveCount(13);
+
+    // make sure the first row is selected
+    await expect(modalOptions.nth(0)).toBeChecked();
+
+    const optionsCount = await modalOptions.count();
```

Comment:
You've already hardcoded the number of options in line 137, so let's also use the hardcoded value instead of calling `.count()`